solutions/3583/01.py

```
@@ -0,0 +1,42 @@
+class Solution:
+    def specialTriplets(self, nums: List[int]) -> int:
```

Ruff (0.14.8)

Undefined name `List`

(F821)